cli/install/install.go

```diff
@@ -909,23 +1013,31 @@ func installTarantool(binDir string, incDir string, installCtx InstallCtx,
 			logFile)
 	} else {
 		log.Infof("Downloading tarantool...")
-		err = downloadRepo(search.GitRepoTarantool, tarVersion, path, logFile, installCtx.verbose)
+		if versionFound {
+			err = downloadRepo(search.GitRepoTarantool, tarVersion, path, logFile,
+				installCtx.verbose)
+		} else {
+			err = downloadRepo(search.GitRepoTarantool, "master", path, logFile, installCtx.verbose)
```

Comment:
Need error check. There is no need to do checkout if clone failed.  This is already fixed for tt. And is not fixed here due to code duplication.

Reply:
fixed

Reply:
Not fully understand why logic for 'version found' scenario differs from 'no found'. If error is set on line 1005, log file name will be printed on line 1016. But if downloadRepo fails on 1008, no log file name is printed. Why? Why don't we set err for git checkout command?

Reply:
fixed